### ISSUE TITLE
[suggestion] upload asset to github

### ISSUE THREAD

**Issue by numToStr** [NTBBloodbath/color-converter.nvim#2]
I noticed that you have a >100kb svg file that is used in the readme. It would be a better experience and make your plugin lighter, if you can upload that to GitHub as GitHub already supports file uploads for readme.

**Comment by NTBBloodbath** [comment]
Hey, that's a great idea!

To be honest I don't know how to upload assets without uploading them to the repository. Do you know how to achieve it?

**Comment by numToStr** [comment]
Yes, just edit readme on GitHub and drag and drop image onto that it will start uploading.

**Comment by NTBBloodbath** [comment]
Thank you!

I'll do it right now, I wasn't able to do it before 'cause I wasn't at home :(
